core/build.gradle

```diff
@@ -21,6 +21,21 @@ android {
     }
 }
 
+repositories {
+    jcenter()
+}
+
+dependencies {
+    testCompile "org.scala-lang:scala-library:2.9.1"
+    testCompile "org.scalatest:scalatest_2.9.1:1.8.RC1"
+    testCompile "junit:junit:4.10"
+    testCompile "com.pivotallabs:robolectric:1.1"
+    testCompile "com.novocode:junit-interface:0.8"
```

Comment:
My view on some of these dependencies
![](http://img.pandawhale.com/92040-that-belongs-in-a-museum-gif-I-AhR4.gif)